public/ashby_hq_aqs_builder_output.json

```diff
@@ -0,0 +1 @@
+{"build_type":"api","core_questions":{"title":"Main application","description":null,"questions":[{"attribute":"full_name","description":null,"label":"Name","required":true,"fields":[{"name":"_systemfield_name","selector":null,"type":"input","options":[]}]},{"attribute":"email","description":null,"label":"Email","required":true,"fields":[{"name":"_systemfield_email","selector":null,"type":"input","options":[]}]},{"attribute":"linkedin","description":null,"label":"LinkedIn","required":true,"fields":[{"name":"05acaa26-5e32-496a-be01-6aae72db3e27","selector":null,"type":"input","options":[]}]},{"attribute":"would_you_require_a_visa_sponsorship_to_relocate_to_london","description":null,"label":"Would you require a visa sponsorship to relocate to London?","required":true,"fields":[{"name":"00b30aa4-0b2a-4e60-b5b7-c80e29d938e2","selector":null,"type":"boolean","options":[]}]},{"attribute":"resume","description":null,"label":"Resume","required":true,"fields":[{"name":"_systemfield_resume","selector":null,"type":"upload","options":[]}]},{"attribute":"why_do_you_want_to_join_us","description":null,"label":"Why do you want to join us?","required":true,"fields":[{"name":"0e089a26-b553-494b-aede-1c03030b6285","selector":null,"type":"textarea","options":[]}]}]},"survey_questions":{"title":null,"description":null,"questions":[]}}
```

Comment:
Can you use pretty_generate for all of these JSON files please going forward. Please notify me once done and I will review. Can we also having these all sit in a folder within public so they're together please. /public/jsons/XXX

Reply:
@patanj101 - one addition to this morning - can you update this to that it's in the output I'm expecting

i.e.:

{
 XXX
}

Reply:
old files ... i just removed them